SerialPrograms/Source/PokemonLZA/Inference/PokemonLZA_DialogDetector.cpp

```diff
@@ -211,15 +211,15 @@ bool NormalDialogDetector::process_frame(const ImageViewRGB32& frame, WallClock
 FlatWhiteDialogDetector::FlatWhiteDialogDetector(Color color, VideoOverlay* overlay)
     : m_color(color)
     , m_overlay(overlay)
-    , m_top(0.267838, 0.784156, 0.467618, 0.019531)
+    , m_bottom(0.265, 0.930, 0.465, 0.020)

```

Comment:
Btw, if you want to easily get box coordinates, you can use the Box Draw program. So if you're wondering why all the new programs have 6 digits of precision for all box coordinates it's because we're all being lazy and using Box Draw to create them for us.